app/routes.js

```diff
@@ -1,43 +1,119 @@
-const express = require('express');
+const express = require("express");
 
-module.exports = (controller) => {
+module.exports = controller => {
     const createApp = () => {
         return express();
     };
 
-    const setup = (app) => {
+    const setup = app => {
         app.use(express.json());
-        app.get('/', (req, res) => {
-            res.send("Welcome to Instagram! <br> Check /api/v1/users/[userId] to view users by ID.");
+
+
+        app.get("/", (req, res) => {
+            res.send(
+                "Welcome to Instagram! <br> Check /api/v1/users/[userId] to view users by ID."
+            );
         });
 
-        app.post('/api/v1/users/', (req, res) => {
-            return controller.createUser(req.body)
+
+        app.get("/api/v1/users/", (req, res) => {
+            return controller
+                .findAllUsers(req)
```

Comment:
here you are using your controller functions that wraps the store, it probably make more sense to create a method getAllUser in the controller that uses getFindUsers from the store. Anyway i don't like the name of this function sin when you want to ALL you're actually not Finding o Searching anything

Reply:
```
app.get("/api/v1/users/", (req, res) => {
            return controller
                .getAllUsers(req)
```
I've changed the naming on this to getAllUsers